integration-tests/testkit/flow.ts

```diff
@@ -50,9 +50,10 @@ export function createOrganization(input: CreateOrganizationInput, authToken: st
                 slug
                 owner {
                   id
-                  organizationAccessScopes
-                  projectAccessScopes
-                  targetAccessScopes
+                  role {
+                    id
+                    permissions
+                  }
```

Comment:
We do not use the legacy scopes anymore, the permissions now always originate from the role.